velox/type/StringView.h

```diff
@@ -251,7 +252,7 @@ struct StringView {
 
   // We rely on all members being laid out top to bottom . C++
   // guarantees this.
-  uint32_t size_;
+  int32_t size_;
```

Comment:
what would a negative value here mean? Should we keep this as unsigned? We're already checking it is positive in the constructor.

Reply:
Its because size_ is assigned len's value before the check and if its uint another implict cast will happen. I can ofcourse change VELOX_CHECK to use len in the constructor if this should be uint. 
Note that we wont be ever able to construct a StringView object eitherway with negative size_